main.go

```diff
@@ -231,8 +231,13 @@ func wrappedMain() int {
 
 func cliConfigFile() (string, error) {
 	mustExist := true
-	configFilePath := os.Getenv("TERRAFORM_CONFIG")
-	if configFilePath == "" {
+	var configFilePath string
+
+	if val, ok := os.LookupEnv("TF_CLI_CONFIG_FILE"); ok {
+		configFilePath = val
+	} else if val, ok := os.LookupEnv("TERRAFORM_CONFIG"); ok {
+		configFilePath = val
+	} else {
```

Comment:
I would suggest still checking is the value is empty.

```
	mustExist := true

	configFilePath := os.Getenv("TF_CLI_CONFIG_FILE")
	if configFilePath == "" {
		configFilePath = os.Getenv("TERRAFORM_CONFIG")
	}

	if configFilePath == "" {
		var err error
		configFilePath, err = ConfigFile()
		mustExist = false
```

Reply:
The line `var configFilePath string` will set the variable to `""` which is why I didn't explicitly check for `""` again. How about something like this?

```
if val, ok := os.LookupEnv("TF_CLI_CONFIG_FILE"); ok {
  configFilePath = val
} else if val, ok := os.LookupEnv("TERRAFORM_CONFIG"); ok {
  configFilePath = val
} else {
  if configFilePath == "" {
    var err error
    configFilePath, err = ConfigFile()
    mustExist = false

    if err != nil {
      log.Printf(
        "[ERROR] Error detecting default CLI config file path: %s",
        err)
    }
  }
}
```
Or should I just consolidate that into an `else if`?

Reply:
Ah I see where the logic can be flawed. One would be able to export an environment variable of `""` and it would then skip the check for empty. Fixing now.

Reply:
Indeed 👍 And I personally try to avoid using `else` or `else if` unless really needed as most of the time I think it makes reading the code less nice/easy to read.